zhazam/Modules/Loading/LoadingViewController.swift

```diff
@@ -8,25 +8,28 @@
 
 import UIKit
 
-class LoadingViewController: UIViewController {
+final class LoadingViewController: UIViewController {
     
-    @IBOutlet weak var nameLabel: UILabel!
-    @IBOutlet weak var percentageLabel: UILabel!
+    @IBOutlet weak private var percentageLabel: UILabel!
     
     override func viewDidLoad() {
         super.viewDidLoad()
-        generateRandomPercents(from: Int.random(in: 1...4) * 10)
+        generateFakeLoading(from: Int.random(in: 1...4) * 10)
     }
     
-    func generateRandomPercents(from number: Int) {
+    private func generateFakeLoading(from number: Int) {
         if number < 100 {
-            let randomNumber = number + Int.random(in: 0...(100-number)/10) * 10
-            percentageLabel.text = randomNumber == 100 ? "\(randomNumber)." : "0\(randomNumber)."
-            let time = Double.random(in: 0.5...1)
-            DispatchQueue.main.asyncAfter(deadline: .now() + time) {
-                self.generateRandomPercents(from: randomNumber)
+            let randomNumber = number + Int.random(in: 0...10-number/10)*10
+            setPercentage(number: randomNumber)
+            let delayTime = Double.random(in: 0.5...1)
+            DispatchQueue.main.asyncAfter(deadline: .now() + delayTime) {
+                self.generateFakeLoading(from: randomNumber)
             }
         }
         //push to next vc
     }
+    
+    private func setPercentage(number: Int) {
```

Comment:
Method name doesn't give clear understanding of what is going to be set. Is it going to alter an instance variable or text of some UI components?

Reply:
done, renamed to setPercentageLabel